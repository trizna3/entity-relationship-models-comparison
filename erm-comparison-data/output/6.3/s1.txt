teams-hockey_teams
matches-matches
rounds-rounds
points-players;matches;points
players-players
REBIND_NARY_ASSOCIATION:S-players-matches-points
