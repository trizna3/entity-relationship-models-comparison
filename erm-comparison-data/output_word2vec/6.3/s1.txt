rounds-rounds
teams-players;matches;points
players-players
points-points
matches-matches
REBIND_NARY_ASSOCIATION:S-players-matches-points
