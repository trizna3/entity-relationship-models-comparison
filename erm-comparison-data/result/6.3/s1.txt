Mapping 4/5 pairs, 1 additional pairs.
Missing pairs: 
 points-points
Additional pairs: 
 points-players;matches;points

Mapping  0/0 transformations, 1 additional transformations.
Missing transformations: -
Additional transformations: 
 rebind_nary_association:s-players-matches-points
