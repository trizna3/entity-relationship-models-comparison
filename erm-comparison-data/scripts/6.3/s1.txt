entity set hockey_teams {
};

entity set players {
	"name"
	"surname"
	"role"
};

entity set matches {
	"winner"
};

entity set points {
	"goals"
	"assists"
	"minute"
	"part"
};

entity set rounds {
	"team_1"
	"team_2"
};


association between 
{
	hockey_teams ONE 
	matches MANY
}{};

association between 
{
	hockey_teams ONE 
	matches MANY
}{};

association between 
{
	hockey_teams ONE 
	players MANY
}{};

association between 
{
	matches ONE 
	rounds MANY
}{};

association between 
{
	players MANY
	matches ONE
	points MANY
}{};