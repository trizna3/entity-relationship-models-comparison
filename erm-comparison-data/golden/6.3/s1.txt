teams-hockey_teams
players-players
matches-matches
points-points
rounds-rounds